src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -81,11 +81,43 @@ public ImmutableList<FileModel> LookupByUid(string uid)
             }
         }
 
+        public MarkupResult MarkupMultiple(string markdown, FileAndType ft)
```

Comment:
why not return a list of markupresult since you already do a split in this method.